lib/std/os/linux.zig

```diff
@@ -3593,11 +3593,12 @@ pub const dl_phdr_info = extern struct {
 };
 
 pub const CPU_SETSIZE = 128;
-pub const cpu_set_t = [CPU_SETSIZE / @sizeOf(usize)]usize;
+pub const CPU_SETLEN = CPU_SETSIZE / @sizeOf(usize);
```

Comment:
`CPU_SETLEN` is not part of `man 3 CPU_SET`. Either remove it or explain what it does.